packages/bpk-mixins/src/mixins/_typography.scss

```diff
@@ -929,10 +929,11 @@ $bpk-spacing-v2: true;
 ///   }
 
 @mixin bpk-table__cell--head {
-  background-color: $bpk-color-sky-gray-tint-07;
-  font-weight: normal;
+  background-color: $bpk-canvas-contrast-day;
   text-align: left;
 
+  @include bpk-label-1;
```

Comment:
Small change after chatting with Adam about the design of the table. This means the header table text will be bold